adflow/pyADflow.py

```diff
@@ -3282,8 +3282,18 @@ def setSurfaceCoordinates(self, coordinates, groupName=None):
         )
         self.mesh.setSurfaceCoordinates(meshSurfCoords)
 
-    def setAeroProblem(self, aeroProblem, releaseAdjointMemory=True):
-        """Set the supplied aeroProblem to be used in ADflow"""
+    def setAeroProblem(self, aeroProblem, releaseAdjointMemory=True, printInfo=True):
```

Comment:
Is there a need to set `printInfo` on every `setAeroProblem`? I would prefer this to be controlled through the options, similar to other print variables (with a more descriptive name though), and not having to pass this through a stack of functions.

Reply:
Sorry for the delay on this.  
I can see that turning stdout printing on/off for various things might be useful in some cases. Controlling the `alpha` using an option is perhaps a bit excessive. I do think though the BCData warnings should probably not be turned off in general, unless you really have a good reason to, as they most often indicate that something is not set properly or potential issues. Overall, there is probably a small demand for creating an option for the end user to disable the BC data warnings. 

However, setting user available option issue aside, I think the static `True/.true.` arguments to functions in various places (both python and fortran) are not great and make the code harder to read and less maintainable. The same functionality can be achieved by making (and exposing in python) a boolean module variable (see [the below comment](https://github.com/mdolab/adflow/pull/307/files#r1304459506) for the variable discussion) that controls the BCData warnings. This variable is then simply set (and reset) when calling the `setAeroProblem` in the mphys wrapper, or alternatively, in the `_setAeroProblemData` if you prefer using the `printInfo` keyword. I think this will make the code more maintainable, as there is no need to carry/set flags in multiple places. 
Let me know if this is unclear, and we can discuss, and perhaps others have further suggestions.